### DISCUSSION TITLE
I can not download it

### DISCUSSION THREAD

**Discussion by lLUCIAN0** [rocksdanister/lively#1098] [Q&A]
Hello!
I've been trying to download it for a while but it tells me that I don't have authorization to download.

**Comment by rocksdanister** [comment]
Try the Windows Store version of Lively and also maybe share a screenshot of this message?

**Reply by lLUCIAN0** [reply]
> Try the Windows Store version of Lively and also maybe share a screenshot of this message?

I tried to download it but the download does not start via the microsoft store
Edit: It can be downloaded from another platform, because I am having errors, and the download does not start

I leave the image sample as is but the download never starts.
https://i13.servimg.com/u/f13/19/81/07/80/1313110.jpg

**Reply by rocksdanister** [reply]
Is this your personal or school/work system?

**Reply by lLUCIAN0** [reply]
Hi @rocksdanister 
I have the question of when I have to restart or shut down my computer. I have to place my background again... And so on every time I do something with the pc.

Is there a way that the moving image is always there, despite a restart or something else?
-I have the netbook for personal use-
